spirekey/e2e/page-objects/pages/example-connect.page.ts

```diff
@@ -40,7 +40,8 @@ export class ExampleConnectPage {
       .context()
       .waitForEvent('page', (p) => p.url().startsWith('http://localhost:1337'));
 
-    await this.connectButton.click();
+    await this.page.getByRole('button', { name: 'Connect' }).waitFor();
```

Comment:
why are we not reuseing the connect button that is defined?
```ts
await this.connectButton.waitFor()
await this.connectButton.click()
```

Reply:
i was playing to find a solution why things broke